plansys2_bt_actions/test/behavior_tree/Move.cpp

```diff
@@ -32,7 +32,7 @@ Move::Move(
   const BT::NodeConfig & conf)
 : plansys2::BtActionNode<test_msgs::action::Fibonacci>(xml_tag_name, action_name, conf)
 {
-  rclcpp_lifecycle::LifecycleNode::SharedPtr node;
+  rclcpp::Node::SharedPtr node;
```

Comment:
This is very sensible. Currently, these BT nodes are used in action performers, which are cascade_lifecycle...

Reply:
I understand the need for the main nodes to be lifecycle nodes. However, the node within the BTActionNode does not have to be a lifecycle node because the action is executed when the ActionExecutorClient instructs it to. In fact, the advantages of lifecycle nodes (configure, activate, etc.) are never utilized in the BTActionNode.

A node client inside the BTActionNode works the same way: 
https://github.com/PlanSys2/ros2_planning_system/compare/rolling...grupo-avispa:ros2_planning_system:improve_bt_actions#diff-d7c4293a0e6ee3223f93bd6fcd115bb5fe9b52f2049ac5e266010e9a7c47fba8R78-R79